### ISSUE TITLE
"High Profile Level 4.2" missing parameter (PS3/PS4)

### ISSUE THREAD

**Issue by Sami32** [UniversalMediaServer/UniversalMediaServer#1043]
While i looked at PlayStation renderers i saw that they was incompletes, but the most important part was a missing implementation of a parameter for _High Profile Level 4.2_ that make unnecessary transcoding for 4.2 profile (or giving error with a 4.2 superior profile if >4.1 profile was set to `true`).

**Comment by SubJunk** [comment]
Yeah it would probably be good to have a renderer setting like MaximumH264Level instead of the existing 4.1 one

**Comment by SubJunk** [comment]
This has been implemented for a long time with `H264LevelLimit` and `H265LevelLimit`
